tendrl/monitoring_integration/objects/definition/monitoring_integration.yaml

```diff
@@ -76,4 +76,13 @@ namespace.monitoring:
         status:
       value: clusters/$integration_id/nodes/$node_id
       metric: [clusters.$integration_id.nodes.$node_name]
+    AlertTypes:
+      enabled: True
+      attrs:
+        types:
+          help: "A Dict of integration to types of alerts the integration handles"
+          type: Dict
```

Comment:
Why dict?
I think list is sufficient

Reply:
ya as per out discussion i had changed this as list